scripts/barclamp_install.rb

```diff
@@ -54,10 +54,13 @@ def usage()
 
 usage if ARGV.length < 1
 
-tmpdir = "/tmp/bc_install-#{Process.pid}-#{Kernel.rand(65535)}"
+tmpdir = "/tmp/component_install-#{Process.pid}-#{Kernel.rand(65535)}"
```

Comment:
I know it's not your fault, but can we use some real solution instead of a hacky thing on this line (mktemp)?